test/fast/FastTokenFacet.test.ts

```diff
@@ -293,7 +284,7 @@ describe('FastTokenFacet', () => {
       await issuerMemberToken.mint(1_000_000, 'ERC20 Tests');
       // Transfer tokens from address zero to alice and bob.
       await Promise.all([alice, bob].map(
-        async ({ address }) => governedToken.transferFrom(ZERO_ADDRESS, address, 100)
+        ({ address }) => issuerMemberToken.transferFrom(ZERO_ADDRESS, address, 100)
```

Comment:
is aync implied?

Reply:
You don't need it for a `Promise.all` as long as your function body returns a promise :)